README.md

```diff
@@ -26,6 +26,44 @@ Useful info:
   
 - https://spark.apache.org/docs/latest/running-on-kubernetes.html
   
+- Livy API documentation: https://livy.incubator.apache.org/docs/latest/rest-api.html
+
+1. Build the spark and the livy docker image by filling in your docker registry in the `acrbuild.sh` script in the `DockerImage/spark` as well as `DockerImage/livy` image
```

Comment:
We should ensure that all instances of <your-container-registry> are replaced in the yaml files.

Reply:
I replaced the image names with `<your-docker-registry>` and searched for occurences of the old image. Didn't find more of them.